### ISSUE TITLE
Mimic BGB palette color blending

### ISSUE THREAD

**Issue by urbster1** [Eiyeron/lsdpatch#38]
It would be really great if the palette editor would show colors more similarly to how they are produced in BGB - see below for example. The palette is completely readable in the editor, but in BGB it is bright and garish.
![image](https://user-images.githubusercontent.com/5685593/47159686-68477c00-d2bc-11e8-87bc-808635e49146.png)

**Comment by Eiyeron** [comment]
I haven't found the exact formulas for game boy's or BGB colors. I don't remember if the colors are already tweaked to mimick a Gameboy's.

Note: I'd probably need some help to reverse engineer all the possible combinaisons unless I find the formulas. I saw some once lost in a forum yet i need to find them.

**Comment by urbster1** [comment]
Yeah - I need to check this too. Odds are the colors appear different on
CGB, SP-101, and GB Player. BGB has its own color settings options so it
could be a BGB issue. Will do research and report further.

**Comment by Eiyeron** [comment]
From what I remember the CGB does an awful job at blending, every channel impacts the other ones in a small way, enough to not get a yellow when pushing red and green channels to full intensity.
